hwy/ops/arm_sve-inl.h

```diff
@@ -2678,6 +2678,92 @@ HWY_API VFromD<D> Reverse8(D d, const VFromD<D> v) {
 HWY_SVE_FOREACH_UI(HWY_SVE_REVERSE_BITS, ReverseBits, rbit)
 #undef HWY_SVE_REVERSE_BITS
 
+// ------------------------------ Block insert/extract/broadcast ops
+#ifdef HWY_NATIVE_BLOCKDFROMD
+#undef HWY_NATIVE_BLOCKDFROMD
+#else
+#define HWY_NATIVE_BLOCKDFROMD
+#endif
+
+template <class D>
+using BlockDFromD =
```

Comment:
Would it make sense to define this in shared-inl.h for targets except RVV?

Reply:
Already moved the BlockDFromD implementation for non-RVV targets over to shared-inl.h